core/src/main/scala/org/apache/spark/internal/config/Status.scala

```diff
@@ -70,4 +70,11 @@ private[spark] object Status {
       .version("3.0.0")
       .booleanConf
       .createWithDefault(false)
+
+  val DISK_STORE_DIR_FOR_STATUS =
+    ConfigBuilder("spark.appStatusStore.diskStore.dir")
```

Comment:
Do we have multiple configurations under the prefix, `spark.appStatusStore.diskStore.`?

Reply:
for now, no other config starts with `spark.appStatusStore.diskStore`

Reply:
If there is no other config, Apache Spark community's configuration naming guide is not to introduce a namespace by removing `.`. In this case,
```
- spark.appStatusStore.diskStore.dir
+ spark.appStatusStore.diskStoreDir
```

Reply:
done. thanks for the guide